projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/DeploymentService.java

```diff
@@ -106,9 +106,27 @@ private void validateFieldsCanBePatched(
         && !mergedDeployment.getGitCommitSha().equals(oldDeployment.getGitCommitSha())) {
       throw new ApiConstraintError(
           "job_version",
-          "same as current job version when using PATCH request.",
+          "same as the current job version when using PATCH request.",
           mergedDeployment.getGitCommitSha(),
-          "Use PUT HTTP request to change job version.");
+          "Use POST HTTP request to change job version.");
+    } else if (mergedDeployment.getPythonVersion() != null
```

Comment:
nit: no need of else if (complications) when exiting the code block with throw; may simply use ifs, it's more readable.

Reply:
Good point. Refactored, thank you for the review